src/components/Environment/ApplicationEnvironmentCards.tsx

```diff
@@ -74,58 +89,43 @@ const ApplicationEnvironmentCard: React.FC<ApplicationEnvironmentCardProps> = ({
 };
 
 type ApplicationEnvironmentCardsProps = {
-  onSelect?: (selectedId: string) => void;
+  onSelect?: (environmentId: string) => void;
+  selectedEnvironment?: string;
+  cardsExpanded: boolean;
 };
 
 export const ApplicationEnvironmentCards: React.FC<ApplicationEnvironmentCardsProps> = ({
+  selectedEnvironment,
   onSelect,
+  cardsExpanded,
 }) => {
-  const [cardsExpanded, setCardExpanded] = React.useState<boolean>(true);
   const [environments, loaded] = useSortedEnvironments();
 
   return (
-    <Flex direction={{ default: 'column' }} grow={{ default: 'grow' }}>
-      <Flex>
-        <FlexItem>
-          <b>Environments:</b>
-          {'  '}
-          <OutlinedHelpPopperIcon
-            heading="Application environments"
-            content="View components and their settings as deployed to environments. Component updates can be promoted between environments. Additional environments can be added via the workspace."
-          />
-        </FlexItem>
-        {loaded && environments.length ? (
-          <FlexItem align={{ default: 'alignRight' }}>
-            <Button
-              onClick={() => setCardExpanded((e) => !e)}
-              variant="link"
-              isInline
-              style={{ textDecoration: 'none' }}
-            >
-              {cardsExpanded ? 'Collapse' : 'Expand'}
-            </Button>
-          </FlexItem>
-        ) : null}
-      </Flex>
-      <Flex alignItems={{ default: 'alignItemsCenter' }}>
-        {loaded ? (
-          environments?.length ? (
-            environments.map((env) => (
-              <FlexItem key={env.metadata.uid}>
-                <ApplicationEnvironmentCard
-                  environment={env}
-                  isExpanded={cardsExpanded}
-                  onSelect={() => onSelect && onSelect(env.metadata.name)}
-                />
-              </FlexItem>
-            ))
-          ) : (
-            <ApplicationEnvironmentCardsEmptyState />
-          )
+    <Flex alignItems={{ default: 'alignItemsCenter' }} flexWrap={{ default: 'nowrap' }}>
+      {loaded ? (
+        environments?.length ? (
+          environments.map((env, index) => (
+            <React.Fragment key={env.metadata.uid}>
+              {isPositionedEnvironment(env, environments.slice(0, index)) ? (
+                <div className="application-environment-cards__env-arrow">
+                  <ArrowRightIcon />
+                </div>
+              ) : null}
+              <ApplicationEnvironmentCard
+                environment={env}
+                isExpanded={cardsExpanded}
+                isSelected={env.metadata.name === selectedEnvironment}
+                onSelect={() => onSelect && onSelect(env.metadata.name)}
```

Comment:
we can use same displayName in the heading below like we have in card title.

![image](https://user-images.githubusercontent.com/9964343/193847990-bab6552d-5dc8-4fc8-9538-5e7825fe37a1.png)


```suggestion
                onSelect={() => onSelect && onSelect(env.spec.displayName)}
```

Reply:
Agreed, I think it would be better to use display name here as well.